README.md

```diff
@@ -1,11 +1,57 @@
-# README
+# sceptre-json-resolver
 
-Add your resolver readme here. Remember to include the following:
+A Sceptre resolver to serialize and deserialize json.
 
-- Tell people how to install it (e.g. pip install ...).
-- Be clear about the purpose of the resolver, its capabilities and limitations.
-- Tell people how to use it.
-- Give examples of the resolver in use.
+## Motivation
 
-If you have any questions or encounter an issue
-[please open an issue](https://github.com/Sceptre/project/issues/new)
+There are use cases where you may want to pass in either a string
+or a json object to a cloudformation or sceptre_user_data parameter.
+This simple resolver can take some json and serialize it or deserialize
+it before passing it into a cloudformation parameter or a
+scepter_user_data parameter.
+
+## Installation
+
+To install directly from PyPI
+```shell
+pip install sceptre-json-resolver
+```
+
+To install from this git repo
+```shell
+pip install git+https://github.com/Sceptre/sceptre-json-resolver.git
+```
+
+## Usage/Examples
+
+```yaml
+parameters|sceptre_user_data:
+  <name>: !from_json [ <string> ]
+
+parameters|sceptre_user_data:
+  <name>: !to_json [ <json object> ]
+```
+
+## Examples
+
+Take some json object serialize it to a string then pass it to a parameter:
+```yaml
+parameters:
+   myparam: !to_json [{"key": "value"}]
+```
+
+Take a string deserialize it to a json object then pass it to a parameter:
+```yaml
+sceptre_user_data:
+  myparam: !from_json ['{"key": "value"}']
```

Comment:
I think you'll need to clarify somewhere that it expects a 1-item list parameter. I know it's reflected in your examples, but it's not _explained_. That's weird enough that I feel like it probably ought to receive some mention.

Reply:
done